rust-toolchain

```diff
@@ -1 +0,0 @@
-1.51.0
```

Comment:
Can you reinstate this file, and rust-nightly-toolchain? They're used (only) to define the toolchains used in CI.

Reply:
Sure thing, I added them back